experimental/oak_baremetal_launcher/src/main.rs

```diff
@@ -18,51 +18,57 @@
 
 use anyhow::Context;
 use clap::Parser;
-use crosvm::Crosvm;
+use instance::LaunchedInstance;
+use instance_crosvm::CrosvmInstance;
+use instance_qemu::QemuInstance;
 use oak_baremetal_communication_channel::{
     client::{ClientChannelHandle, RequestEncoder},
     schema,
 };
-use qemu::Qemu;
 use std::{
     fs,
     io::{BufRead, BufReader},
     os::unix::net::UnixStream,
     path::PathBuf,
 };
 use tokio::signal;
-use vmm::{Params, Vmm};
 
-mod crosvm;
+mod instance;
+mod instance_crosvm;
+mod instance_qemu;
 mod lookup;
-mod qemu;
 mod server;
-mod vmm;
 
-#[derive(clap::ArgEnum, Clone, Debug, PartialEq)]
+/// Parameters used for launching VM instances
+#[derive(Parser, Clone, Debug, PartialEq)]
+pub struct VmParams {
```

Comment:
Just from a code layout perspective I think the params should still be in (the moral equivalent of) `instance.rs`.

Reply:
I typically find it helpful to keep the structs defining CLI args in the head of the main file, but fine, will change, don't feel strongly about this.

Reply:
On a second thought, I think we should keep it here. Having the params be in one place is also what xtask does. Similarly they reuse the `path_exists` function in this file for validation. We could import it around, but I don't think it'd really make things clearer.